### PR TITLE
add support for X-Forwarded-Prefix for path striping reverse proxy

### PR DESCRIPTION
**Pull request by bhugot**
It's a missing feature for the reverse proxy usage

### CODE REVIEW

**Comment by heldersepu** [comment]
A unit test for this will be nice...

**Comment by bhugot** [comment]
Done

File: Swashbuckle.Tests/Swagger/DefaultRootUrlResolverTests.cs [hunk 1]
```diff
@@ -26,10 +26,10 @@ public void It_provides_scheme_host_and_port_from_x_forwarded()
             request.Headers.Add("X-Forwarded-Proto", "https");
             request.Headers.Add("X-Forwarded-Host", "acmecorp.org");
             request.Headers.Add("X-Forwarded-Port", "8080");
-
+            request.Headers.Add("X-Forwarded-Prefix", "/api");
             var rootUrl = SwaggerDocsConfig.DefaultRootUrlResolver(request);
 
-            Assert.AreEqual("https://acmecorp.org:8080", rootUrl);
+            Assert.AreEqual("https://acmecorp.org:8080/api", rootUrl);
```
Comment (heldersepu) [inline Swashbuckle.Tests/Swagger/DefaultRootUrlResolverTests.cs]: Nice job!

**Comment by bhugot** [comment]
Easy one

**Comment by bhugot** [comment]
@heldersepu Do you know if you are going to do a new release soon?

**Pull request merged**